packages/sil-governance/src/routes/dashboard/components/Proposal/CollapsingData/Description.tsx

```diff
@@ -0,0 +1,22 @@
+import { Typography } from "medulas-react-components";
+import React from "react";
+import { ellipsify } from "ui-logic";
+
+const DESC_MAX_LENGTH = 180;
+
+interface Props {
+  readonly fullDescription: string;
+  readonly expanded: boolean;
+}
+
+const Description = ({ fullDescription, expanded }: Props): JSX.Element => {
+  const description = expanded ? fullDescription : ellipsify(fullDescription, DESC_MAX_LENGTH) + " ";
```

Comment:
Why the extra space? Why only in one of the two cases?

Reply:
To leave some space for the "Read more". In the "Read less" case isn't needed because it's on a new line. The difference is shown in the screenshots

Reply:
Makes sense, thanks